apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -827,20 +827,36 @@ internal_block_pending(_Config) ->
                 <<"BlockWithTxsHashes">>,
```

Comment:
I see that two lines above (at line 825) `get_pending_block()` re-attempts. I would therefore recommend removal of `timer:sleep(100)` at line 824, as that makes it more likely for a block having already been mined (even if high difficulty for high inter-block interval set).